torch/testing/_internal/common_utils.py

```diff
@@ -1422,12 +1422,12 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *,
         return torch.randint(0, 2, size, device=device, dtype=dtype)
 
     if dtype is torch.uint8:
-        low = math.floor(0 if low is None else max(low, 0))
-        high = math.ceil(10 if high is None else min(high, 10))
+        low = math.ceil(0 if low is None else max(low, 0))
```

Comment:
Why change the floor/ceils here?

Reply:
`rsqrt`'s domain is `(0, inf]`

So I have kept the domain for `low=1e-15, high=inf`

With current evaluation,
`low = math.floor(0 if low is None else max(low, 0))`
Low turns to be `0` which is actually out of domain. That is why we use `ceil` instead. 

Similarly for high, if for a function with domain `[0, 8)`,
If we keep `high=7.999`,
`high = math.ceil(10 if high is None else min(high, 10))`
High will be `8` which is actually out of domain. That is why we use `floor`.

So this change is mostly to handle these exclusive range correctly.